components/datadog/apps/fakeintake/fargateFakeintakeService.go

```diff
@@ -79,6 +86,90 @@ func NewECSFargateInstance(e aws.Environment, option ...fakeintakeparams.Option)
 			return nil, err
 		}
 
+		ca, err := acmpca.NewCertificateAuthority(e.Ctx, namer.ResourceName("ca"), &acmpca.CertificateAuthorityArgs{
+			Type: pulumi.StringPtr("ROOT"),
+			CertificateAuthorityConfiguration: acmpca.CertificateAuthorityCertificateAuthorityConfigurationArgs{
+				KeyAlgorithm:     pulumi.String("RSA_4096"),
+				SigningAlgorithm: pulumi.String("SHA512WITHRSA"),
+				Subject: acmpca.CertificateAuthorityCertificateAuthorityConfigurationSubjectArgs{
+					CommonName: e.CommonNamer.DisplayName(64, pulumi.String("fakeintake CA")),
+				},
+			},
+			RevocationConfiguration: acmpca.CertificateAuthorityRevocationConfigurationArgs{
+				CrlConfiguration: acmpca.CertificateAuthorityRevocationConfigurationCrlConfigurationArgs{
+					Enabled: pulumi.BoolPtr(false),
+				},
+				OcspConfiguration: acmpca.CertificateAuthorityRevocationConfigurationOcspConfigurationArgs{
+					Enabled: pulumi.Bool(false),
+				},
+			},
+			// UsageMode:                   pulumi.StringPtr("SHORT_LIVED_CERTIFICATE"),
```

Comment:
I think we can remove the comment.

Reply:
Fixed: https://github.com/DataDog/test-infra-definitions/pull/463/commits/e10a97f9b0aa648e0796e5957e3d73b61c584c9a